soc/arm64/qemu_cortex_a55/Kconfig.defconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>
+# Copyright 2022 NXP
```

Comment:
If these files are new (.defconfig, .ld, .soc, etc.) why do you add this copyright:  "Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>"?
Are taken from somewhere else?

Reply:
Yup, they are. Please see https://github.com/zephyrproject-rtos/zephyr/tree/main/soc/arm64/qemu_cortex_a53. As a rule I added the copyright from the initial file and if I had to modify anything I also added the NXP copyright.

Reply:
yep, I got it latter  - I commented below.
We all use, sometimes, other files as examples, but these are new, even if similar, I think you should let only NXP copyright.